shoop/admin/modules/orders/views/create.py

```diff
@@ -223,6 +219,7 @@ def _handle_source_data(self, request):
     def _handle_create(self, request):
         state = json.loads(request.body.decode("utf-8"))["state"]
         order = create_order_from_state(state, creator=request.user)
+        order_complete_viewed.send(sender=self, order=order, request=request)
```

Comment:
No typo. order_creator_finished is called in OrderCreator:create_order. The problem this solves is that orders created through admin should send this signal too for the order notifications.